cephfs/admin/workflow_test.go

```diff
@@ -158,6 +158,8 @@ func TestWorkflow(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, sinfo)
 
+	time.Sleep(500 * time.Millisecond) // is there a race?
```

Comment:
AFAIK, CephFS snap creation is not completely synchronous. So this can be expected.

Reply:
Thanks! It's been pretty fast, but this will hopefully prevent future flakiness.